providers/song-controls-front.js

```diff
@@ -1,18 +1,43 @@
 const { ipcRenderer } = require("electron");
 
-let videoStream = document.querySelector(".video-stream");
+let video = document.querySelector("video");
+
 module.exports = () => {
-    ipcRenderer.on("playPause", () => {
-        if (!videoStream) {
-            videoStream = document.querySelector(".video-stream");
-        }
+    ipcRenderer.on("playPause", (_e, toPlay) => playPause(toPlay));
+};
+module.exports.playPause = playPause;
 
-        if (videoStream.paused) {
-            videoStream.play();
-        } else {
-            videoStream.yns_pause ?
-                videoStream.yns_pause() :
-                videoStream.pause();
+function playPause(toPlay = undefined) {
+    if (!checkVideo()) return;
+    
+    switch(toPlay) {
+        case undefined: 
+            video.paused ? play() : pause();
+            break;
+        case true:
+            play(); 
+            break;
+        case false:
+            pause();
+    }
+
+    function play() {
+        video.play();
+    }
+    
+    function pause() {
```

Comment:
nit: it might not be necessary to define the 2 inner functions if they are a one-liner and only used once!

Reply:
`I agree that ``play()` is unnecessary, but `pause()` is called in 2 different places so it has a point..
(I originally added the play() function only because pause() was there - but I removed it as requested)